release_notes.md

```diff
@@ -1,13 +1,6 @@
-### Release notes
-
-<!-- Please add your release notes in the following format:
-- My change description (#PR)
--->
-- Update Java Worker Version to [2.19.0](https://github.com/Azure/azure-functions-java-worker/releases/tag/2.19.0)
-- Add support for new FeatureFlag `EnableAzureMonitorTimeIsoFormat` to enable iso time format for azmon logs for Linux Dedicated/EP Skus. (part of #7864)
-- Update PowerShell worker to 4.0.4175 (sets defaultRuntimeVersion to 7.4 in worker.config.json)
-- Fixing default DateTime bug with TimeZones in TimerTrigger (#10906)
-- Add support for the release channel setting `WEBSITE_PlatformReleaseChannel` and use this value in extension bundles resolution.
-- Bug fix for platform release channel bundles resolution casing issue and additional logging (#10921)
-- Fix race condition that leads the host to initialize placeholder (warmup) function in Linux environments (#10848)
-- Update Python Worker Version to [4.36.1](https://github.com/Azure/azure-functions-python-worker/releases/tag/4.36.1)
+### Release notes
+
+<!-- Please add your release notes in the following format:
+- My change description (#PR)
+-->
+- Add JitTrace Files for v4.39

```

Comment:
Do we need to include this in release notes? It is an implementation detail, not customer facing.

Reply:
Since we are creating the hotfix just to add the JIT trace file, I will keep the entry for this release. However, going forward, I agree that this is an implementation detail and does not need to be included in the release notes.